src/dlgTriggerEditor.cpp

```diff
@@ -419,34 +419,56 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
     showTimersAction->setStatusTip(tr("Show Timers"));
+    showTimersAction->setToolTip(QStringLiteral("%1 (%2)").arg(tr("Show Timers"), tr("Ctrl+4")));
     connect(showTimersAction, &QAction::triggered, this, &dlgTriggerEditor::slot_show_timers);
 
-    QAction* viewScriptsAction = new QAction(QIcon(QStringLiteral(":/icons/document-properties.png")), tr("Scripts"), this);
-    viewScriptsAction->setStatusTip(tr("Show Scripts"));
-    connect(viewScriptsAction, &QAction::triggered, this, &dlgTriggerEditor::slot_show_scripts);
-
     QAction* viewKeysAction = new QAction(QIcon(QStringLiteral(":/icons/preferences-desktop-keyboard.png")), tr("Keys"), this);
     viewKeysAction->setStatusTip(tr("Show Keybindings"));
+    viewKeysAction->setToolTip(QStringLiteral("%1 (%2)").arg(tr("Show Keybindings"), tr("Ctrl+5")));
     connect(viewKeysAction, &QAction::triggered, this, &dlgTriggerEditor::slot_show_keys);
 
     QAction* viewVarsAction = new QAction(QIcon(QStringLiteral(":/icons/variables.png")), tr("Variables"), this);
     viewVarsAction->setStatusTip(tr("Show Variables"));
+    viewVarsAction->setToolTip(QStringLiteral("%1 (%2)").arg(tr("Show Variables"), tr("Ctrl+6")));
     connect(viewVarsAction, &QAction::triggered, this, &dlgTriggerEditor::slot_show_vars);
 
+    QAction* viewActionAction = new QAction(QIcon(QStringLiteral(":/icons/bookmarks.png")), tr("Buttons"), this);
+    viewActionAction->setStatusTip(tr("Show Buttons"));
+    viewActionAction->setToolTip(QStringLiteral("%1 (%2)").arg(tr("Show Buttons"), tr("Ctrl+7")));
+    connect(viewActionAction, &QAction::triggered, this, &dlgTriggerEditor::slot_show_actions);
+
+
+    QAction* viewErrorsAction = new QAction(QIcon(QStringLiteral(":/icons/errors.png")), tr("Errors"), this);
+    viewErrorsAction->setStatusTip(tr("Show/Hide the errors console in the bottom right of this editor."));
+    viewErrorsAction->setToolTip(QStringLiteral("%1 (%2)").arg(tr("Show/Hide errors console"), tr("Ctrl+8")));
+    connect(viewErrorsAction, &QAction::triggered, this, &dlgTriggerEditor::slot_viewErrorsAction);
+
+    QAction* viewStatsAction = new QAction(QIcon(QStringLiteral(":/icons/view-statistics.png")), tr("Statistics"), this);
+    viewStatsAction->setStatusTip(tr("Generate a statistics summary display on the main profile console."));
+    viewStatsAction->setToolTip(QStringLiteral("%1 (%2)").arg(tr("Generate statistics"), tr("Ctrl+9")));
+    connect(viewStatsAction, &QAction::triggered, this, &dlgTriggerEditor::slot_viewStatsAction);
+
+    QAction* showDebugAreaAction = new QAction(QIcon(QStringLiteral(":/icons/tools-report-bug.png")), tr("Debug"), this);
+    showDebugAreaAction->setStatusTip(tr("Show/Hide the separate Central Debug Console - when being displayed the system will be slower."));
+    showDebugAreaAction->setToolTip(tr("Show/Hide Debug Console (Ctrl+0) -> system will be <b><i>slower</i></b>."));
```

Comment:
Is this positioning of the short-cut optimum - perhaps it should be at the end?
![image](https://user-images.githubusercontent.com/6163092/122688782-3083f580-d216-11eb-89c8-f9b89813859e.png)

Reply:
Yes, it is after the name/description proper and before the random usage info, that should probably not be part of tool-tip to begin with, and is doubled in status text as well.

Reply:
Actually the hint that it gives about slowing things down ***is*** appropriate - at least for the first few times the user sees it. https://www.trychameleon.com/blog/why-tooltips-are-terrible-and-why-you-should-use-them

Reply:
Anyhow - for a consistent UI I think the tool-tip needs to go to the end. And also for consistency it is probably better to put all of them in a separate paragraph, e.g.:
```suggestion
    showDebugAreaAction->setToolTip(QStringLiteral("<p>%1</p><p>%2</p>").arg(tr("Show/Hide Debug Console -> system will be <b><i>slower</i></b>."), tr("(Ctrl+0)", "Shortcut to toggle Debug console from a profile's editor window."));
```

Reply:
> Actually the hint that it gives about slowing things down _**is**_ appropriate - at least for the first few times the user sees it. https://www.trychameleon.com/blog/why-tooltips-are-terrible-and-why-you-should-use-them

The link talks about introduction of features, seems related to #2823 maybe